keps/sig-node/20190131-runas-groupid.md

```diff
@@ -0,0 +1,44 @@
+---
+kep-number: 0
+title: RunAsGroup support in PodSpec and PodSecurityPolicy
+authors:
+  - "@krmayankk"
+owning-sig: sig-node
+participating-sigs:
+  - sig-auth
+reviewers:
+  - "@tallclair"
+  - "@mrunalp"
+approvers:
+  - "@liggitt"
+  - "@derekwaynecarr"
+editor: TBD
+creation-date: 2017-06-21
+last-updated: 2017-09-14
+status: implementable
+---
+
+# RunAsGroup support in PodSpec and PodSecurityPolicy
+
+https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/runas-groupid.md
+
+## Table of Contents
+
+* [Graduation Criteria](#graduation-criteria)
+
+* [Implementation History](#implementation-history)
+
+
+## Graduation Criteria
+
+- Publish Test Results from Master Branch of Cri-o To http://prow.k8s.io [#72253](https://github.com/kubernetes/kubernetes/issues/72253)
+- Containerd and CRI-O tests included in k/k CI [#72287](https://github.com/kubernetes/kubernetes/issues/72287)
+- Make CRI tests failures as release informing
```

Comment:
Are there any validation changes that need to be made before going to beta?

Reply:
no, validation was done properly in alpha to not require changes on promotion